app/views/user_email_mailer/response_invite.html.erb

```diff
@@ -63,6 +63,21 @@
           margin: 0 auto;
           padding: 8px;
       }
+      .field_empty {
```

Comment:
I would recommend to use naming convention ( you can read about it here - https://sass-lang.com/styleguide/code ) using dashes "lalala-lalala". Underlines are usable, but not in you case.